src/App.css

```diff
@@ -469,3 +475,47 @@ button:enabled:hover {
   color: #1c2334;
   text-decoration: underline;
 }
+
+/* Search Form */
+.search {
+  align-items: center;
+  background-color: white;
+  border-radius: 0.5rem;
+  display: flex;
+  flex-direction: row;
```

Comment:
with `display: flex;` `row` is the default value, so you don't have to declare that.

Reply:
Thanks 😄 ! the `flex-direction` was removed 👍 .